src/Communication/SAF.Communication.PubSub.Cde/Publisher.cs

```diff
@@ -8,14 +8,20 @@
 using System.Threading.Tasks;
 using SAF.Common;
 using SAF.Communication.Cde;
+using SAF.Communication.Cde.ConnectionTypes;
 using SAF.Communication.PubSub.Interfaces;
 
 namespace SAF.Communication.PubSub.Cde
 {
+    /// <summary>
+    /// Publishes messages (using <see cref="SubscriptionRegistry"/>) to all registered subscribers via
+    /// <see cref="DefaultComLine"/>.<br/>
```

Comment:
Its via `ComLine` and not via `DefaultComLine`. `ComLine` is the abstract base class for `DefaultComLine` and `AdvancedComLine` (used with CDE multi scoping environments).